_targets.R

```diff
@@ -24,19 +26,29 @@ major_ion_names = c("Chloride","Sodium")
 # Define hydrologic event types in harmonized WQP data to exclude
 omit_wqp_events <- c("Spill","Volcanic action")
 
-# Define USGS specific conductance parameter codes
-SpC_pcodes <- c("00095","90095","00094","90096") 
+# Define all USGS specific conductance parameter codes
+pcodes_select <- c("00095","90095","00094","90096") 
 
-# Define minor HUCs (hydrologic unit codes) that make up the DRB (HUC04 = 0204)to use in calls to dataRetrieval functions
+# Define USGS specific conductance parameter code to use for continuous data
+parameter <- "00095"
```

Comment:
related to comment above, are there other codes for instantaneous data?

Reply:
My understanding is that "00095" is the code used for in situ SC monitoring. The table below is from the USGS [report](https://nc.water.usgs.gov/reports/tm1D3/pdf/TM1D3.pdf) "Guidelines and standard procedures for continuous water-quality monitors." I also did a query within the DRB and "00095" is the only code returned for instantaneous ("uv") or daily summary ("dv") sites. By the way, did you know that pcodes are getting phased out as part of NWIS/Aquarius modernization? 

![Wagner2006_Table23](https://user-images.githubusercontent.com/8785034/144513241-6f8fde5e-c06d-4efa-b7ff-cbd5b0e45e01.png)

Reply:
Thanks! I did not know that pcodes are going away. Should we be adapting to the new format for this work? And what is the new format? We can discuss outside of this PR if that makes more sense

Reply:
I learned about it from the DRB data harmonization group - I think the plan is to transition to "observed properties and units" with a rollout in FY21/22. That group didn't seem to know enough about it to change their data workflows and I'm guessing current systems will be supported for awhile. So I suggest we shouldn't worry about altering our workflows at this stage but it's good to know that it's (maybe) coming...

Reply:
FYI more details [in this Teams conversation ](https://teams.microsoft.com/l/message/19:e320b241dd91479fba9b83c5bf9f6d99@thread.skype/1638537022782?tenantId=0693b5ba-4b18-4d7b-9341-f32f400a5494&groupId=2651eb2e-8cf1-4caa-a084-0a11facc1d36&parentMessageId=1638537022782&teamName=GS-WMA%20IIDD%20Staff&channelName=General&createdTime=1638537022782)on this transition

Reply:
Thanks, @jzwart! So it sounds like discrete data ("qw") pulls will be impacted first, but that there will be a ~2 year transition period from pcodes to observed properties (OPs). I assume this plan will also apply to unit values when the time comes.

Reply:
Yes that's my understanding.